kf_task_fhir_etl/target_api_plugins/entity_builders/proband_status.py

```diff
@@ -0,0 +1,99 @@
+"""
+Builds FHIR Observation resources (https://www.hl7.org/fhir/observation.html)
+from rows of tabular participant broband status data.
+"""
+from abc import abstractmethod
+
+from kf_lib_data_ingest.common import constants
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+from kf_task_fhir_etl.target_api_plugins.entity_builders import Patient
+from kf_task_fhir_etl.common.utils import not_none, drop_none, yield_resource_ids
+
+# http://hl7.org/fhir/ValueSet/observation-status
+status_code = "final"
+
+# http://terminology.hl7.org/CodeSystem/v2-0136
+value_coding = {
+    constants.COMMON.TRUE: {
+        "system": "http://terminology.hl7.org/CodeSystem/v2-0136",
+        "code": "Y",
+        "display": "Yes",
+    },
+    constants.COMMON.FALSE: {
+        "system": "http://terminology.hl7.org/CodeSystem/v2-0136",
+        "code": "N",
+        "display": "No",
+    },
+}
+
+
+class ProbandStatus:
+    class_name = "proband_status"
+    api_path = "Observation"
+    target_id_concept = None
+    service_id_fields = None
+
+    @classmethod
+    def get_key_components(cls, record, get_target_id_from_record):
+        patient_id = not_none(get_target_id_from_record(Patient, record))
+        proband_status = not_none(record[CONCEPT.PARTICIPANT.IS_PROBAND])
+        assert proband_status in ["True", "False"]
+
+        return {
+            "code": "http://snomed.info/sct|85900004",
+            "subject": f"{Patient.api_path}/{patient_id}",
+        }
+
+    @classmethod
+    def build_entity(cls, record, get_target_id_from_record):
+        study_id = record[CONCEPT.STUDY.TARGET_SERVICE_ID]
+        proband_status = record[CONCEPT.PARTICIPANT.IS_PROBAND]
+
+        entity = {
+            "resourceType": cls.api_path,
+            "id": get_target_id_from_record(cls, record),
+            "meta": {
+                "profile": [f"http://hl7.org/fhir/StructureDefinition/{cls.api_path}"],
+                "tag": [{"code": study_id}],
+            },
+            "identifier": [
+                {
+                    "use": "official",
+                    "system": "https://kf-api-dataservice.kidsfirstdrc.org/participants?is_proband=",
```

Comment:
```suggestion
                    "system": "https://kf-api-dataservice.kidsfirstdrc.org/participants?is_proband",
```

Reply:
I'm leaving one comment here but it applies to all the occurrences of this observation. I'm wondering why these `system` fields that have a url + query parameter are appended with the `=` sign?

Reply:
Let me create another ticket to discuss this.